builder/test-integration/spec/hypriotos-image/docker_spec.rb

```diff
@@ -1,12 +1,17 @@
 require 'spec_helper'
 
 describe package('docker-engine') do
+  it { should_not be_installed }
+end
+
+describe package('docker-ce') do
   it { should be_installed }
 end
 
-describe command('dpkg -l docker-engine') do
-  its(:stdout) { should match /ii  docker-engine/ }
-  its(:stdout) { should match /17.05.0~ce-0~raspbian-jessie/ }
+describe command('dpkg -l docker-ce') do
+  its(:stdout) { should match /ii  docker-ce/ }
+  its(:stdout) { should match /17.09.0~ce-0/ }
```

Comment:
Can we check here for raspbian package?

Reply:
We could check for `17.09.0~ce-0~raspb`, that's the string which gets displayed.

Reply:
I tried `dpkg -l docker-ce | grep raspbian`
and got `ii  docker-ce      17.09.0~ce-0~raspbian armhf        Docker: the open-source application container engine`
Maybe the output is only truncated for the terminal.

Reply:
Interesting, this output depends on the screen width of the terminal. So, I can test with `17.09.0~ce-0~raspbian` and the test runs successfully. Will commit this change now.